data/human/hails.txt

```diff
@@ -1096,6 +1096,50 @@ phrase "friendly civilian"
 		"deserves to be put down!"
 		"has to be replaced!"
 
+phrase "friendly civilian"
+	word
+		"Some"
+		"Some of my"
+	word
+		" "
+	word
+		"passengers"
+		"guests"
+		"VIP passengers"
+	word
+		" "
+	word
+		"ate asparagus"
+		"ate garlic"
+		"ate beans"
+	word
+		". "
+	word
+		"I left them at the spaceport. Let them reek in someone else's ship."
+		"The heads are still stinking, badly."
+		"I'm happy they are leaving on the next stop."
```

Comment:
Some of these sentences reference the passengers as being current, and others refer to former passengers, which causes problems for the tense of the first sentence.

Reply:
I'll make those all future tense (to match current passengers).